### DISCUSSION TITLE
Skywalking failed to submit log

### DISCUSSION THREAD

**Discussion by chenqinwei** [apache/skywalking#8023] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

Phenomenon: the springboot project uses the logback method to report logs to skywalking 8.7. Logback.xml and agent.config are configured. After the springboot project is started, the access interface does not report errors, and the skywalking background does not report errors, but there are warnings. You can find that the interface is called in the skywalking page, but the log list is empty

Environment: the springboot project and agent are on the same machine, and skywalking 8.7 is deployed on another machine
![企业微信截图_16353207402171](https://user-images.githubusercontent.com/88814006/139022382-df687323-2db3-499e-a515-1b1b6f42e88a.png)

![企业微信截图_16353207118134](https://user-images.githubusercontent.com/88814006/139022330-e6beafbf-f7c8-4d79-9530-fc4abe6dd751.png)
![企业微信截图_16353207635085](https://user-images.githubusercontent.com/88814006/139022439-5e7874d5-78c6-4457-a63d-8d078f883b22.png)
![企业微信截图_16353207936924](https://user-images.githubusercontent.com/88814006/139022517-f1e3cbaf-c330-414b-a56b-35764b3940e5.png)
![企业微信截图_16353208703025](https://user-images.githubusercontent.com/88814006/139022746-48c946bb-6011-4902-be24-5d767324878f.png)

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
The warning logs are irrelative. And I can't see clear reason of what is going on.

**Reply by chenqinwei** [reply]
I want to report the business system log to skywalking and display it on the front page, but now the page display is blank. Can you help me see the problem?

My operation is divided into three steps:

1. Introduce APM toolkit trace and APM toolkit logback-1. X into the business system

2. Collect logs in logback.xml

3. Configure agent.config and specify the address and port reported by grpc
